src/components/Books.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Books = () => (
+  <div>
+    <h4>List of books</h4>
+    <li>
+      <button type="button">Remove</button>
+    </li>
```

Comment:
- [ ] You should not use a `<li/>` tag outside `<ul/>` tags, kindly change it for another and more semantic tag 🚀